task_2/8.js

```diff
@@ -0,0 +1,26 @@
+// Создать объект human со следующими возможностями:
+// 1. При записи значения в свойство fullName имя и фамилия должны записываться в отдельные свойства firstName и lastName
+// 2. При чтении значения fullName оно должно получаться из объединения firstName и lastName
+// 3. При задании значения свойства dateOfBirth должно так же устанавливаться свойство age в зависимости от разницы года рождения и текущего года
+
+const human = Object.create({}, {
+  fullName: {
+    get: function() {
+      return `${this.lastName || ''} ${this.firstName || ''}`.trim()
+    },
+    set: function(val) {
+      [this.lastName, this.firstName] = val.split(' ')
+    }
+  },
+  dateOfBirth: {
+    get: function() {
+      return new Date(`${this.monthOfBirth}.${this.dayOfBirth}.${this.yearOfBirth}`)
```

Comment:
Попробуйте использовать [конструктор даты](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date) в таком формате:
```suggestion
      return new Date(this.yearOfBirth, this.monthOfBirth, this.dayOfBirth)
```